docs/book/src/reference/watching-resources/secondary-resources-owned-by-other-controllers.md

```diff
@@ -0,0 +1,78 @@
+# Watching Secondary Resources Owned by Other Controllers or Projects
+
+In some cases, a controller may need to watch and react to changes in resources
+that are not `Owned` by it, but are created and managed by another controller.
+
```

Comment:
Here, I think it would be useful to list common use cases:

* when the primary resource *references* something
* when the primary object is named after some other object (?)

Reply:
We kind of explain that in another place.
We have the Primary Resource , everything that is required to that works on the cluster would be a Secondary resource .. but I will give a look and see if I can improve with your suggestions thx.